src/ray/gcs/gcs_server/gcs_placement_group_scheduler.cc

```diff
@@ -410,11 +415,6 @@ void GcsPlacementGroupScheduler::OnAllBundleCommitRequestReturned(
     return;
   }
 
-  // Acquire bundle resources from gcs resources manager.
-  const auto &committed_bundle_locations =
-      lease_status_tracker->GetCommittedBundleLocations();
-  AcquireBundleResources(committed_bundle_locations);
```

Comment:
NOTE: As I mentioned in the PR's description, the `AcquireBundleResources` is moved before `AcquireResources`, I think it has no impact.

Reply:
We should make sure when the prepare is failed, acquired resources are properly released. Can you check (1) if we have cpp unit tests for them (2) if not, can you add them? I think we need tests on these scenarios

1. Resources are acquired, and prepare is failed because the node is dead.
2. Resources are acquired and prepare is failed because the resources were not available on that node.
3. Resources are acquired and commit is failed because the node is dead. 

And in these cases, acquired resources must be released again.

Reply:
Okey, I will add these unit tests.